### PR TITLE
Move Proxy TPM (ptpm) to its own directory

### PR DESCRIPTION
**Pull request by shjala**
Just a clean up, move Proxy TPM (ptpm) to its own directory.

### CODE REVIEW

File: pkg/vtpm/Dockerfile [hunk 1]
```diff
@@ -40,8 +40,11 @@ RUN cp /usr/local/lib/libtss2* /out/usr/lib/
 # build tpm2-tools, it is needed by ptpm
 WORKDIR /tpm2-tools
 ADD --keep-git-dir=true https://github.com/tpm2-software/tpm2-tools.git#5.5 /tpm2-tools
-COPY patch/patch-tpm2-tools.diff .
-RUN patch -p1 < patch-tpm2-tools.diff
+COPY patches/ /patches/
```
Comment (OhmSpectator) [inline pkg/vtpm/Dockerfile:43]: Have we also renamed the directory from "patch" to "patches"?

**Comment by shjala** [comment]
Yetus is complaining over unsupported syntax : 
```
Running mini-yetus
./tools/mini-yetus.sh -f  
[+] Running mini-yetus
[+] No source branch specified. Using the main branch: master
[+] No destination branch specified. Using the current branch: move.ptpm
[+] Running yetus on the changes...
[+] Full results:
/home/shah/shah-dev/eve/pkg/vtpm/Dockerfile:33:25:hadolint: invalid flag: --keep-git-dir
[+] Results stored in /tmp/yetus.d6JoEPfOY0/yetus-output
```

as for the lint, I think one-line commit message is sufficient in this case.

**Review by OhmSpectator — APPROVED** [review]
Let's run some tests

**Comment by OhmSpectator** [comment]
> as for the lint, I think one-line commit message is sufficient in this case.

okaaay =(

**Comment by OhmSpectator** [comment]
> /home/shah/shah-dev/eve/pkg/vtpm/Dockerfile:33:25:hadolint: invalid flag: --keep-git-dir

I googled for it and bumped into @deitch's old comment) https://github.com/hadolint/hadolint/issues/609#issuecomment-1296835076

**Comment by shjala** [comment]
> > /home/shah/shah-dev/eve/pkg/vtpm/Dockerfile:33:25:hadolint: invalid flag: --keep-git-dir
> 
> I googled for it and bumped into @deitch's old comment) [hadolint/hadolint#609 (comment)](https://github.com/hadolint/hadolint/issues/609#issuecomment-1296835076)

Yup, not the first time I've seen it, my haskell is at basic reading level otherwise would have send a PR.

**Pull request merged**